models/events.js

```diff
@@ -237,10 +238,35 @@ const createEventCode = async (eventCodeData) => {
   }
 };
 
+const getAllEventCodes = async (roomId) => {
```

Comment:
getAllEventCodes function is declared as a const function, but it is using the async and await keywords. This is not allowed, because const functions cannot be asynchronous.

Reply:
As per I know, I think we can use const. You can check in the examples here as well:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/async_function

Please let me know if I'm missing anything.